util/version-outputs.js

```diff
@@ -8,6 +8,9 @@ const  generate = `
   ${chalk.gray('Upgrade to latest version:')}
   $ ${chalk.green('npm i @aurodesignsystem/wc-generator@latest -g')}
 
+  ${chalk.gray('Uninstall previous version(only needed if there is an issue):')}
+  $ ${chalk.green('npm uninstall -g @aurodesignsystem/wc-generator')}
```

Comment:
would this require privilege escalation in some environments?

Reply:
To uninstall a global npm? What would be the reason?

Reply:
Some of our onboarding install scripts use sudo or an escalated PowerShell prompt to install dependencies, and I don't believe the local user has the ability by default to remove items from the global npm folder. I think it's safe to approve this with awareness that this could come up